mmv1/templates/terraform/examples/region_network_firewall_policy_rule.tf.erb

```diff
@@ -0,0 +1,67 @@
+resource "google_network_security_address_group" "basic_regional_networksecurity_address_group" {
+  name        = "address-group"
```

Comment:
This needs to be a variable so it gets prefixed with our normal test prefix

Reply:
Good call.. Will change here!